packages/logrotate/build

```diff
@@ -10,6 +10,12 @@ make
 make install
 
 
+# Create the module file
+module="$PKG_PATH/dcos.target.wants_slave/logrotate.json"
```

Comment:
Yeah, I was thinking about that but then got confused when I saw several `wants_` things in other build files and thought maybe there is some pkgpanda magic going on here :). I'll fix it and rerun the test suite. (There is another mesos build issue that was just resolved in master, so will update my branch.)